### DISCUSSION TITLE
Welcome to XrmToolBox Discussions!

### DISCUSSION THREAD

**Discussion by MscrmTools** [MscrmTools/XrmToolBox#1001] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by daryllabar** [comment]
I feel that Discussions are highly under utilized.

**Reply by rappen** [reply]
💯

**Reply by MscrmTools** [reply]
I do agree but please don't make it our private joke place :D
